spec/sendcloud/operations/sender_addresses_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Sendcloud::Operations::SenderAddressesRequest do
+  let(:default_base_url) { "https://panel.sendcloud.sc/api/v2/" }
+
+  before do
+    configure_client(base_url: default_base_url)
+  end
+
+  describe "#execute" do
+    let(:subject) { described_class.new }
+
+    it "returns a response in the expected format" do
+      VCR.use_cassette("sender_addresses_request/valid_request") do
+        result = subject.execute
+
+        aggregate_failures do
+          expect(subject.response.status).to eq(200)
+          expect(result).to be_instance_of(Array)
+          expect(result.first.keys).to include(:id, :label)
```

Comment:
isnt this also tested below? or do you expect the first item to be different to the one below?

Reply:
No I don't expect any difference, just copy/pasted an existing spec